assignment01/js/myScript1.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready();

```

Comment:
So - you are checking for the document to load here - but you aren't calling or defining a function. The assignment was to create an anonymous function here that sets the name values, appends hobby list items and set the css style for the mcc class. Please let me know if you have questions. We will build on these concepts from here.

Reply:
I just plain missed the point here. For some reason, I was thinking that the whole point of the assignment was just link the javascript library, and to check if the document was loaded. I just figured that the other tasks were refreshers in html. They are not. I've gone back over it.